SPECS-EXTENDED/numatop/numatop.spec

```diff
@@ -59,6 +79,10 @@ autoreconf -ivf
 
 
 %changelog
+* Fri Dec 20 2024 Durga Jagadeesh Palli <v-dpalli@microsoft.com> - 2.4-1
```

Comment:
If it is a spec directly imported from Fedora, please use the original changelog itself.